cocos/2d/assembler/label/font-utils.ts

```diff
@@ -28,7 +28,7 @@ import { ImageAsset, Texture2D } from '../../../asset/assets';
 import { PixelFormat } from '../../../asset/assets/asset-enum';
 import { BufferTextureCopy } from '../../../gfx';
 import { safeMeasureText, BASELINE_RATIO, MIDDLE_RATIO, getBaselineOffset, getSymbolCodeAt } from '../../utils/text-utils';
-import { director, Director } from '../../../game/director';
+import { director, Director, DirectorEvent } from '../../../game/director';
```

Comment:
Should remove Director as it is not used anymore?

Reply:
Yes, I will remove it.

Reply:
Updated.